cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1290,14 +1514,122 @@ instance Show peerConn => ToObject (DebugPeerSelection Socket.SockAddr peerConn)
              ]
 
 instance ToObject (PeerSelectionActionsTrace Socket.SockAddr) where
-  toObject _verb ev =
-    mkObject [ "kind" .= String "PeerSelectionAction"
-             , "event" .= show ev ]
+  toObject _verb (PeerStatusChanged ps) =
+    mkObject [ "kind" .= String "PeerStatusChanged"
+             , "peerStatusChangeType" .= show ps
```

Comment:
Could we provide a json encoding?

Reply:
yes, but needs to be over at `ouroboros-network`, I'm going to leave a TODO

Reply:
It's fine to provide all the orphaned instance here (at least this will be consistent).

Reply:
Got it!

Reply:
Oh I remembered why we can't: because `PeerStatusChangeType` is not exported by `ouroboros-network` :/

Reply:
Leave a todo then, and let's export `PeerStatusChangeType`.